### PR TITLE
Address input target

### PR DESCRIPTION
**Pull request by yuli-ferna**
The required validations: 

OFAC Check:

https://github.com/user-attachments/assets/92a09d6f-8d43-4aba-9852-e15df261c2cf

Address validation:

https://github.com/user-attachments/assets/d791ed19-23e6-4de7-a0e0-65ae5071327f

Disable Manual Route when active Manual address option:

https://github.com/user-attachments/assets/18c17ef7-9607-42b1-b678-2c0b5cae4116

### CODE REVIEW

File: wormhole-connect/src/routes/abstracts/routeAbstract.ts [hunk 1]
```diff
@@ -17,6 +17,16 @@ import {
 import { ParsedRelayerMessage, ParsedMessage } from 'utils/sdk';
 import { TokenPrices } from 'store/tokenPrices';
 
+export const REASON_AMOUNT_TOO_LOW = 'Transfer amount too low';
+export const REASON_MANUAL_ADDRESS_NOT_SUPPORTED =
+  'Manual address not supported';
+export interface AvailableReason {
```
Comment (M-Picco) [inline wormhole-connect/src/routes/abstracts/routeAbstract.ts]: Perhaps rename to `RouteAvailability`

File: wormhole-connect/src/store/transferInput.ts [hunk 2]
```diff
@@ -104,7 +104,7 @@ export type TransferValidations = {
 export type RouteState = {
   name: string;
   supported: boolean;
-  available: boolean;
+  validation: AvailableReason;
```
Comment (M-Picco) [inline wormhole-connect/src/store/transferInput.ts]: Rename to `availability`

File: wormhole-connect/src/views/WalletModal.tsx [hunk 3]
```diff
@@ -226,11 +243,97 @@ function WalletsModal(props: Props) {
     );
   };
 
+  const [address, setAddress] = useState('');
```
Comment (M-Picco) [inline wormhole-connect/src/views/WalletModal.tsx]: Could the manual address logic be bundled up in a new component?

File: wormhole-connect/src/components/QRScanner.tsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
+import React, { ComponentProps, useCallback, useState } from 'react';
+import QrCodeIcon from '@mui/icons-material/QrCode';
+import { IconButton } from '@mui/material';
+import { makeStyles } from 'tss-react/mui';
+import { Scanner } from '@yudiel/react-qr-scanner';
+import Modal from 'components/Modal';
+
+const SCANNER_HEIGHT_PX = 500;
```
Comment (M-Picco) [inline wormhole-connect/src/components/QRScanner.tsx:8]: Any chance of using relative sizes instead of absolutes?

Comment (yuli-ferna) [reply wormhole-connect/src/components/QRScanner.tsx:8]: in this case the number is used for attributes like minHeight and maxWidth, so it does not represent problems in responsive

File: wormhole-connect/src/views/WalletModal.tsx (hunk 3 above)
Comment (yuli-ferna) [reply wormhole-connect/src/views/WalletModal.tsx]: Done!

File: wormhole-connect/src/store/transferInput.ts (hunk 2 above)
Comment (yuli-ferna) [reply wormhole-connect/src/store/transferInput.ts]: Done!

File: wormhole-connect/src/routes/abstracts/routeAbstract.ts (hunk 1 above)
Comment (yuli-ferna) [reply wormhole-connect/src/routes/abstracts/routeAbstract.ts]: Done!

**Review by sebastianscatularo — APPROVED** [review]
LGTM 🚀

**Comment by agodlevska** [comment]
Checked on https://preview.portalbridge.com/859350d189/. 
A bug regarding wallet address validation has been created - [Incorrect error notification](https://github.com/XLabs/portal-bridge-ui/issues/964). 
A bug regarding manual route disabling has been created - [Manual route remains disabled](https://github.com/XLabs/portal-bridge-ui/issues/966).
A bug regarding incompatibility wallet and network - [Cosmos and Solana networks with MetaMask](https://github.com/XLabs/portal-bridge-ui/issues/967)

**Comment by tsadovska** [comment]
Checked on https://preview.portalbridge.com/859350d189/.
Issues created:

- https://github.com/XLabs/portal-bridge-ui/issues/968
- https://github.com/XLabs/portal-bridge-ui/issues/969
- https://github.com/XLabs/portal-bridge-ui/issues/970

**Comment by tsadovska** [comment]
Checked on https://preview.portalbridge.com/859350d189/.
Issues created:
* https://github.com/XLabs/portal-bridge-ui/issues/977
* https://github.com/XLabs/portal-bridge-ui/issues/979
* https://github.com/XLabs/portal-bridge-ui/issues/980
* https://github.com/XLabs/portal-bridge-ui/issues/985
* https://github.com/XLabs/portal-bridge-ui/issues/987
* https://github.com/XLabs/portal-bridge-ui/issues/988

**Comment by tsadovska** [comment]
Checked on https://preview.portalbridge.com/859350d189/.
Issues created:
* https://github.com/XLabs/portal-bridge-ui/issues/989
* https://github.com/XLabs/portal-bridge-ui/issues/990
* https://github.com/XLabs/portal-bridge-ui/issues/991

**Comment by tsadovska** [comment]
Checked on https://preview.portalbridge.com/859350d189/.
Issues created:
* https://github.com/XLabs/portal-bridge-ui/issues/992
* https://github.com/XLabs/portal-bridge-ui/issues/994

**Comment by tsadovska** [comment]
Checked on https://preview.portalbridge.com/859350d189/.
Issues created:
* https://github.com/XLabs/portal-bridge-ui/issues/998
* https://github.com/XLabs/portal-bridge-ui/issues/999
* https://github.com/XLabs/portal-bridge-ui/issues/1000
* https://github.com/XLabs/portal-bridge-ui/issues/1001
* https://github.com/XLabs/portal-bridge-ui/issues/1002

**Pull request merged**